packages/amplify-e2e-tests/src/schema-api-directives/tests/predictions-usage.ts

```diff
@@ -55,16 +55,21 @@ async function uploadImageFile(projectDir: string) {
   }) as any;
 
   const bucketName = amplifyMeta.storage[storageResourceName].output.BucketName;
-
-  const fileStream = fs.createReadStream(imageFilePath);
-  const uploadParams = {
-    Bucket: bucketName,
-    Key: imageKey,
-    Body: fileStream,
-    ContentType: 'image/jpeg',
-    ACL: 'public-read',
-  };
-  await s3Client.upload(uploadParams).promise();
+  try {
+    const fileStream = fs.createReadStream(imageFilePath);
+    const uploadParams = {
+      Bucket: bucketName,
+      Key: imageKey,
+      Body: fileStream,
+      ContentType: 'image/jpeg',
+      ACL: 'public-read',
+    };
+    await s3Client.upload(uploadParams).promise();
+  } catch (err) {
+    if (err.code !== 'AccessControlListNotSupported') {
+      throw err;
+    }
+  }
```

Comment:
how is this helping?
if test is not sensitive to this upload then why don't we just remove this step altogether?

Reply:
Currently this test is not validating anything after this file upload step. There is a backlog item to fix this.
This test is still useful to make sure a GraphQL schema with predictions directive can be compiled and pushed to cloud.

Reply:
I agree here - this is a duplicate test. Can we just delete this test? I think this adds more confusion if this doesn't end up getting "fixed" in the future than it does anything else